pkg/apis/integreatly/v1alpha1/rhmi_types.go

```diff
@@ -27,6 +27,7 @@ var (
 
 	InstallationTypeWorkshop InstallationType = "workshop"
 	InstallationTypeManaged  InstallationType = "managed"
+	InstallationTypePDS InstallationType = "rhpds"
```

Comment:
We can remove this line and just use `InstallationTypeManaged`, it's synonymous with an RHPDS deploy

Reply:
`InstallationTypeWorkshop`, I mean

Reply:
Ok, sounds good.